src/tools/mongoc-stat.c

```diff
@@ -29,6 +29,11 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
+#define BSON_INSIDE
+#define MONGOC_INSIDE
```

Comment:
These wrapping macros are not needed for `mlib/cmp.h`

Reply:
OK. I am very confused. I am nearly 100% certain that the first time I tried without the macros and the build failed. I had contemplated adding them, but I wasn't sure if it was OK. I asked Ezra and he showed me an example of adding the macros in a test for a case where the direct inclusion was needed. In any event, I removed them and ran a new patch build, and it worked. So I assume that something changed in the underlying code between when I first observed the failure and now. Any event, a commit is incoming that removes these.